spyder/plugins/application/plugin.py

```diff
@@ -121,6 +126,25 @@ def on_mainwindow_visible(self):
             container.give_updates_feedback = False
             container.check_updates(startup=True)
 
+    def update_debug_logs(self):
+        """Create an action for each lsp and debug log file."""
+        self.debug_logs_menu.clear()
+        if self.is_plugin_available(Plugins.Editor):
+            load = self.get_plugin(Plugins.Editor).load
+        else:
+            def load():
+                pass
+
+        debug_logs = []
+        files = glob.glob(os.path.join(get_conf_path('lsp_logs'), '*.log'))
+        files.append(os.environ['SPYDER_DEBUG_FILE'])
+        for f in files:
+            action = self.create_action(
+                f, f, triggered=load)  # TODO: add triggered
+            action.setData(f)
+            debug_logs.append(action)
+        add_actions(self.debug_logs_menu, debug_logs)
```

Comment:
`add_actions` is a low level function that in general should not be used. The way to add actions to menus is by using the PluginMainContainer/PluginMainWidget API, like this:

```python
for action in debug_logs:
    self.add_item_to_menu(
        action,
        menu=self.debug_logs_menu,
    )
```